core/src/main/resources/org/apache/spark/ui/static/streaming-page.css

```diff
@@ -63,4 +63,4 @@ span.expand-input-rate {
 
 tr.batch-table-cell-highlight > td {
   background-color: #D6FFE4 !important;
-}
+}
```

Comment:
Could you avoid the file content change? If you use `git mv`, it will generate a clean history.